### PR TITLE
Cosmetic changes

### PR DESCRIPTION
**Pull request by waldo1001**

### CODE REVIEW

**Review by JeremyVyska — APPROVED** [review]

File: content/docs/BestPractices/CustomTelemetry/index.md [hunk 1]
```diff
@@ -31,11 +31,11 @@ Therefore, signal must be treated as any other API
 
 To make it easy for the consumer of telemetry to work with the data, please
 
-- use PascalCasing. This makes all fields in Application Insights look the same (signal logged through the AL LogMessage method will have "al" prefixed to dimension names.
-- never use fields/custom dimension keys with spaces or special characters. This makes the KQL queries so much easier to write
+- use **PascalCasing**. This makes all fields in Application Insights look the same (signal logged through the AL LogMessage method will have "al" prefixed to dimension names.
+- **Don't use special characters or spaces** for fields/custom dimension keys. This makes the KQL queries so much easier to write
```
Comment (TheDoubleH) [inline content/docs/BestPractices/CustomTelemetry/index.md:35]:
```suggestion
- **Do n0t use special characters or spaces** for fields/custom dimension keys. This makes the KQL queries so much easier to write
```

**Review by TheDoubleH — COMMENTED** [review]
Suggestion: Change "Don't" to "Do not"

**Review by TheDoubleH — CHANGES_REQUESTED** [review]
Grammar

**Comment by waldo1001** [comment]
@TheDoubleH - you're right - I approved your changes ;-)

**Review by TheDoubleH — APPROVED** [review]

**Review by JeremyVyska — APPROVED** [review]

File: content/docs/BestPractices/api-page/index.md [hunk 2]
```diff
@@ -162,13 +162,13 @@ It is common use to give certain fields a more describing name. Some examples ar
...
 * displayName for field Name
 
-## Default fields
-An API should at least expose these fields:
+## Mandatory fields
+These fields should always be part of the API Page:
 
 * SystemId
-  * This field should be exposed wit the name ```id```
+  * This field should be exposed wit the name ```id```
```
Comment (JesperSchulz) [inline content/docs/BestPractices/api-page/index.md:169]: wit -> with

File: content/docs/BestPractices/CustomTelemetry/index.md [hunk 3]
```diff
@@ -65,13 +65,13 @@ end;
...
+Telemetry data includes information about the object that emitted the telemetry signal. It's recommended to call Session.LogMessage() **from within the object** that causes a situation that you want to have telemetry for. That will make it easier to analyze where exactly in the code an issue occurred.
 
 Of course it is possible to have a single object as a central place to emit telemetry signals. The telemetry data includes a callstack, so eventually it would be possible to trace back to the exact place where an issue occurred. But that requires a more complicated query, so it would be better to emit telemetry signals right from place in the code where an issue occurred.
 
 ## Candidate data for telemetry
 
-Telemetry must be actionable for the customer. Do not emit signals that they cannot act on (knowing about CPU performance counters on the database is useless if the partner cannot scale the database)
+Telemetry must be **actionable** for the customer. Do not emit signals that they cannot act on (knowing about CPU performance counters on the database is useless if the partner cannot scale the database)
```
Comment (JesperSchulz) [inline content/docs/BestPractices/CustomTelemetry/index.md:74]: Period missing in the end.

**Review by JesperSchulz — APPROVED** [review]

**Pull request merged**